test/common/apiHelpers.js

```diff
@@ -227,10 +230,12 @@ var getUnconfirmedTransactionPromise = node.Promise.promisify(getUnconfirmedTran
 var getUnconfirmedTransactionsPromise = node.Promise.promisify(getUnconfirmedTransactions);
 var getMultisignaturesTransactionPromise = node.Promise.promisify(getMultisignaturesTransaction);
 var getMultisignaturesTransactionsPromise = node.Promise.promisify(getMultisignaturesTransactions);
-var getPendingMultisignaturePromise = node.Promise.promisify(getPendingMultisignature);
+var getPendingMultisignaturesPromise = node.Promise.promisify(getPendingMultisignatures);
+var sendTransactionPromise = node.Promise.promisify(sendTransaction);
```

Comment:
It's already 3 times duplicated

Reply:
No, it is not. They are different function. Of course, they need to be reviewed in the API multisig refactoring.